src/main/resources/policies/werklijst-rechten.rego

```diff
@@ -19,6 +20,11 @@ documenten_inbox {
     recordmanager.rol in user.rollen
 }
 
+default productaanvragen_inbox := false
```

Comment:
Ik vraag ne af of hier een apart recht voor nodig is. Ik zou zelf het documenten_inbox recht hernoemeb naar inbox en deze voor beide inbox lijsten gebruiken.

Reply:
subtask voor aangemaakt #2432